packages/electron/src/config/main.js

```diff
@@ -30,7 +31,7 @@ module.exports.schema = {
     validate: value => typeof value === 'function'
   },
   projectRoot: {
-    defaultValue: () => app.getAppPath(),
+    defaultValue: () => normalizePath(app.getAppPath()),
```

Comment:
I'm not totally sure if this is a good idea — it does fix the problem when testing locally with macOS but maybe doesn't work as well on Windows?

The integration tests still pass so I think it's ok, but the other change does fix the issue so this might just be unnecessary

Reply:
yeah I'm not sure about this for two reasons:

1. it seems unnecessary, seeing as you duplicate the logic to add a trailing slash in the renderer anyway
2. it doesn't normalize the path it it is set in configuration (only the default value)

the only thing you'd miss in the renderer would be resolving any `..`s.

An alternate suggestion: you could call out to `normalizePath` in the `plugin-electron-ipc/lib/config-serializer.js` module?

Reply:
It seems fine if the integration tests are fine, though after a quick check, I think the only context assertions are on custom context. So adding assertions for default context for renderer events would cover everything.

Reply:
Is this not covered by the stacktrace assertions?

Reply:
oh yeah! that should cover it.